intake/src/main/resources/variantProcessor.py

```diff
@@ -43,15 +43,17 @@ def get_engine(self):
 
     def get_is_dichotomous(self, phenotype_name):
         with self.get_engine().connect() as connection:
-            rows = connection.execute(f'SELECT name, dichotomous FROM Phenotypes WHERE name = \"{phenotype_name}\"').all()
+            query = sqlalchemy.text(f'SELECT name, dichotomous FROM Phenotypes WHERE name = \"{phenotype_name}\"')
```

Comment:
The actual fixes. Mainly just this text thing (instead of pegging to sqlalchemy 1.14), and then I added NA as a "null-like" value that can be seen in the intake files.